apps/rush-lib/src/logic/taskRunner/TaskRunner.ts

```diff
@@ -443,4 +443,21 @@ export class TaskRunner {
     }
   }
 
+  /**
+   * Remove trailing blanks, and all middle lines if text is large
+   */
+  private _abridgeTaskReport(text: string): string {
+    const headSize: number = 10;
+    const tailSize: number = 20;
+    const margin: number = 10;
+    const lines: Array<string> = text.split(/\s*\r?\n/).filter(line => line);
+    if (lines.length < headSize + tailSize + margin) {
+      return lines.join(os.EOL);
+    }
+    const amountRemoved: number = lines.length - headSize - tailSize;
+    const head: string = lines.splice(0, headSize).join(os.EOL);
+    const tail: string = lines.splice(-tailSize).join(os.EOL);
+    return `${head}${os.EOL}[...${amountRemoved} lines elided...]${os.EOL}${tail}`;
```

Comment:
Huh. I've never heard the word "elided" before. Would "omitted" be better?

Reply:
Sure, to be honest I did not spontaneously think of it: saw it [there](https://twitter.com/garybernhardt/status/1097307043881308161), and thought for a moment it was more accurate than "omit", but "omit" is fine.
Done.